.github/templates/android_ci_full_workflow.yml.j2

```diff
@@ -0,0 +1,153 @@
+{%- extends "linux_ci_workflow.yml.j2" -%}
+{% import 'common_android.yml.j2' as common_android %}
+{%- set exclude_test = true -%}
+{% block name -%}
+# Template is at:    .github/templates/android_ci_workflow.yml.j2
```

Comment:
This should be android_ci_full_workflow.yml.j2 I think
```suggestion
# Template is at:    .github/templates/android_ci_full_workflow.yml.j2
```

Reply:
yes, thank you!